pkg/services/publicdashboards/service/service.go

```diff
@@ -45,6 +46,7 @@ type PublicDashboardServiceImpl struct {
 }
 
 var LogPrefix = "publicdashboards.service"
+var tracer = otel.Tracer("publishdashboards")
```

Comment:
we've been using the package name here:

```suggestion
var tracer = otel.Tracer("github.com/grafana/grafana/pkg/services/publicdashboards/service")
```